web/src/components/AddPlayerForm.js

```diff
@@ -29,11 +29,11 @@ function AddPlayerForm(props) {
                 
                 <Form.Group className="mb-3">
                     <Form.Label htmlFor='playername'>Name</Form.Label>
-                    <Form.Control type="name" placeholder="taro" ref={nameInputRef} />
+                    <Form.Control required type="name" placeholder="taro" ref={nameInputRef} />
```

Comment:
It's best practice to use `required`. But let's not use this for some practice in the future. Could you roll it back like before?

Reply:
What kind of practices would be?

Reply:
We can practice two things. I've made these issues. 
- add validation check on the server side https://github.com/nishi-ai/softball-register/issues/9
- UI error message https://github.com/nishi-ai/softball-register/issues/10

There would be better ways always like what you did. But it's good idea to practice things without helps from tools one time.